plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcClient.java

```diff
@@ -109,4 +112,33 @@ public void testMetadataWithFloatAndDoubleCol()
                 new JdbcColumnHandle("COL3", JDBC_DOUBLE, DOUBLE),
                 new JdbcColumnHandle("COL4", JDBC_REAL, REAL)));
     }
+
+    @Test
+    public void testCreateSchema()
+    {
+        String schemaName = "mixed case schema";
+        jdbcClient.createSchema(session, schemaName);
+        assertThat(jdbcClient.getSchemaNames(session)).contains(schemaName);
+        jdbcClient.dropSchema(session, schemaName);
+        assertThat(jdbcClient.getSchemaNames(session)).doesNotContain(schemaName);
+    }
+
+    @Test
+    public void testRenameTable()
+    {
+        String schemaName = "test_schema";
+        SchemaTableName oldTable = new SchemaTableName(schemaName, "foo");
+        SchemaTableName newTable = new SchemaTableName(schemaName, "bar");
+        ConnectorTableMetadata tableMetadata = new ConnectorTableMetadata(
+                oldTable,
+                ImmutableList.of(new ColumnMetadata("text", VARCHAR)));
+
+        jdbcClient.createSchema(session, schemaName);
```

Comment:
Could you please consider introducing `TestSchema` similar to `io.trino.testing.sql.TestTable`?

Reply:
`TestTable` requires an executor and executes queries directly. It also doesn't quote identifiers. This test explicitly uses `createSchema` from the jdbc client. I think it's slightly out of scope of this PR.